php/commands/search-replace.php

```diff
@@ -115,17 +119,32 @@ public function __invoke( $args, $assoc_args ) {
 			$table->setRows( $report );
 			$table->display();
 
-			if ( !$dry_run )
+			if ( ! $dry_run ) {
 				WP_CLI::success( "Made $total replacements." );
+			}
 
 		}
 	}
 
-	private static function get_table_list( $args, $network, $all_with_prefix ) {
+	/**
+	 * Retrieve a list of tables from the database.
+	 *
+	 * @global wpdb $wpdb
+	 *
+	 * @param array $assoc_args Array of options passed to this command.
+	 *
+	 * @return array The array of table names.
+	 */
+	private static function get_table_list( $assoc_args ) {
```

Comment:
Would it make more sense to have the default be only WordPress tables, rather than all tables? I ask because that's the current behavior. That would give 4 total options:

* `wordpress` (default)
* `network`
* `all-tables-with-prefix`
* `all-tables`